src/js/router.js

```diff
@@ -3,19 +3,22 @@
 //
 // The express module will automatically attempt to wire in anything with this gradeName into its routing table.
 //
+// When the underlying router is created, its constructor will be passed `options.routerOptions`.  For a list of the
+// supported options, check out [the Express documentation](http://expressjs.com/api.html#router).
+//
 // This implementation is not meant to be used directly.  When wrapping most existing Express routers, you will want
 // to implement your own `router` method and (typically) point your `router` invoker at that.
 //
 // For all other use cases, you should likely start with `requestAwareRouter` and implement your own handler.
 //
-var fluid = fluid || require("infusion");
+var fluid = require("infusion");
 var gpii  = fluid.registerNamespace("gpii");
 fluid.registerNamespace("gpii.express.router");
 
 // Instantiate our router object.  The root gpii.express object will wire everything together
 gpii.express.router.createRouter = function (that) {
     var express         = require("express");
-    that.options.router = express.Router();
+    that.options.router = express.Router(that.options.routerOptions);
```

Comment:
Must not write into a component's options - in future framework releases these will be read-only. Assign it to a top-level that.router instead